eks_buildspec.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.2
```

Comment:
Would you mind walking me through how this gets executed and what this is going to be executed by? Is there a new code build project that essentially creates and manages the eks cluster? If so, when does this actually get run?